snap/snapcraft.yaml

```diff
@@ -150,7 +150,30 @@ apps:
       after:
         - core-config-seed
         - mongo-worker
-
+  device-modbus:
+    command: bin/go-services-wrapper.sh device-modbus
+    daemon: simple
+    plugs: [network, network-bind]
```

Comment:
I assume at some point this will change, but for now is it expected that modbus can only talk to modbus devices over TCP? I think so but just want affirmation...